src/Pact/Parse.hs

```diff
@@ -67,8 +64,8 @@ expr = do
     ]
 {-# INLINE expr #-}
 
-
-number :: P Literal
+-- TODO Consider: http://hackage.haskell.org/package/megaparsec-7.0.0/docs/Text-Megaparsec-Char-Lexer.html#v:float
```

Comment:
why? please provide a specific reason instead of leaving TODOs around.

Reply:
Number parsing is a long-solved problem, I thought it odd that there would be custom code for it here. Although I'm not sure of the relationship between `PactParser` and the rest of the parsing libs used here, so the suggestion might be moot anyway.